networking_monitoring.prolific

```diff
@@ -97,6 +97,99 @@ L: security
 
 ---
 
+Inspect your c2c packets with tcpdump
+
+### What?
+Sometimes a user says "container to container networking is broken! AppA can't talk to AppB". After making sure that they have c2c policies set, the next thing you might do is use tcpdump to inspect the packets.
+
+Tcpdump is a CLI tool that allows you to inspect all of the traffic (not just TCP, despite the name) flowing through your container. This is a great tool to use to debug how far packets are making it, how long the packets are taking to reach their destination, and to look at where in the TCP handshake things are failing.
+
+In this story you are going to look at the packets being sent from Boots to Swiper. Then you'll watch the packets being sent in response.
+
+### How?
+**Setup**
+1. You should have two apps pushed named Boots and Swiper.
+1. There should be no c2c network policies (`cf network-policies` and `cf remove-network-policies --help`).
+
+**Curl swiper from boots**
+1. Get the overlay IPs of boots and swiper.
+ ```
+cf ssh boots -c "env | grep CF_INSTANCE_INTERNAL_IP"
+cf ssh swiper -c "env | grep CF_INSTANCE_INTERNAL_IP"
+ ```
+1. Ssh onto boots.
+ ```
+cf ssh boots
+ ```
+1. Continually try to curl swiper from boots
```

Comment:
Noticing inconsistent use of capitol lettering for the app names in sentences. Would be nice to standardize.